pkgs/top-level/all-packages.nix

```
@@ -28936,5 +28936,7 @@ in
 
   lc3tools = callPackage ../development/tools/lc3tools {};
 
+  security-wordlists = callPackage ../tools/security/wordlists/default.nix {};
```

normal convention to drop the default.nix explicitly
```suggestion
  security-wordlists = callPackage ../tools/security/wordlists {};
```

---

also, I'm not a big fan of new attrs at the end of `all-packages.nix`, usually leads to a lot of merge conflicts.